plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/util/MinioClient.java

```diff
@@ -181,7 +197,12 @@ private void putObject(String bucket, ByteSource byteSource, String targetPath)
     {
         try {
             try (InputStream inputStream = byteSource.openStream()) {
-                client.putObject(bucket, targetPath, inputStream, null, null, null, null);
+                client.putObject(
+                        PutObjectArgs.builder()
+                                .bucket(bucket)
+                                .object(targetPath)
+                                .stream(inputStream, inputStream.available(), -1)
```

Comment:
`InputSteam.available()` != file size.

use `byteSource.size();`

Reply:
We could also do `stream(inputStream, -1, MIN_MULTIPART_SIZE)`  to avoid unnecessarily reading the `byteSource` to get its size.

Reply:
yeah, this is not needed now.